tests/testing.R

```diff
@@ -0,0 +1,16 @@
+#read in the data
```

Comment:
I'm not sure why this code is in `tests/testing.R` instead of case_study 1.  Please move it there and also commit the other case studies.